src/components/index/page-sections/VenturesSection.jsx

```diff
@@ -23,22 +23,22 @@ const VenturesSection = React.forwardRef(({ animationAreaHeight }, ref) => {
       <MobileAndTabletQuery>
         <VenturesSectionContent>
           <Link to="/contact">
-            <GradientButtonBasic
+            <GradientButton
               theme="light"
               widthClass="w-full"
```

Comment:
hier ist nochmal widthClass - evtl könntest du nochmal im Projekt danach suchen und gucken dass das überall raus kommt

Reply:
Done :) 
Bei 1-2 Images habe ich ebenfalls widthClass gefunden - gilt da hier dann auch nicht mehr? 
Habe es jetzt erstmal nur bei den Buttons entfernt

Reply:
Glaube die Images haben die prop noch - sonst einfach in die Komponenten reingucken :D Idealerweise wäre es besser wenns überall einfach className wäre, aber ist glaube ich etwas aufwendig überall zu refactoren